pyccel/epyccel.py

```diff
@@ -181,28 +237,47 @@ def epyccel( inputs, **kwargs ):
 
         # Master process calls epyccel
         if comm.rank == root:
-            fmod      = epyccel_seq( inputs, **kwargs )
-            fmod_path = os.path.abspath(fmod.__file__)
-            fmod_name = fmod.__name__
+            try:
+                mod, fun = epyccel_seq( python_function_or_module, **kwargs )
+                mod_path = os.path.abspath(mod.__file__)
+                mod_name = mod.__name__
+                fun_name = python_function_or_module.__name__ if fun else None
+                success  = True
+            except:
+                exc_info = sys.exc_info()
```

Comment:
Does this collect information about the exception? Would it be useful to also add the information contained in the exception itself?

Reply:
This catches the type, the value and the traceback. `exc_info` is a tuple.